packages/kit/src/runtime/server/index.js

```diff
@@ -23,6 +26,26 @@ export class Server {
 		this.#options = options;
 		this.#manifest = manifest;
 
+		// Since AsyncLocalStorage is not working in webcontainers, we don't reset `sync_store`
+		// in `src/exports/internal/event.js` and handle only one request at a time.
+		if (IN_WEBCONTAINER) {
```

Comment:
Better safe than sorry: should we also check if ALS doesn't exist?

Reply:
The problem on stackblitz is that ASL does exist but doesn't work.